src/bilder/images/edx_notes/files/docker-compose.yaml

```diff
@@ -15,6 +15,17 @@ services:
     # Ref: CONFIG_ROOT
     # https://github.com/openedx/edx-notes-api/blob/master/notesserver/settings/yaml_config.py
     - /etc/docker/compose/edx_notes_settings.yaml:/app/edx-notes-api/edx_notes_api.yml:ro
+  notes_es_create_index:
+    image: 610119931565.dkr.ecr.us-east-1.amazonaws.com/dockerhub/mitodl/openedx-notes:${OPENEDX_RELEASE}
+    env_file:
+    - ./.env
+    depends_on:
+    - notes_api
```

Comment:
The relationship should be inverted. This is intended to be an init container that runs before the app. While we're at it we should also include a django migrate command as part of the init, probably before the ES index.